packages/agoric-cli/src/commands/gov.js

```diff
@@ -279,9 +337,7 @@ export const makeEconomicCommiteeCommand = (_logger, io = {}) => {
       /** @type {Parameters<typeof processOffer>[0]['toOffer']} */
       const toOffer = (agoricNames, current) => {
         const cont = current ? findContinuingIds(current, agoricNames) : [];
-        const votingRight = cont.find(
-          it => it.instance === agoricNames.instance.economicCommittee,
-        );
+        const votingRight = cont.find(it => it.instanceName === opts.instance);
```

Comment:
I'm doing type-matching by eye, and `it.instanceName` doesn't look like the same kind of thing as `opts.instance`. Is this correct?  I couldn't find an actual type declaration, nor any other use of `opts.instance`.

Reply:
`opts` comes from the CLI options so it's this up above,
```js
    .requiredOption(
      '--instance <string>',
      'Committee name under agoricNames.instances',
      String,
      'economicCommittee',
    )
```
That does match the `instanceName` out of the result set.